tests/playwright/Report.md

```diff
@@ -0,0 +1,176 @@
+- [Puppeteer Evaluation](https://www.sohamp.dev/blog/2020-08-24-wmf-gsoc2020/)
+
+
+So, this brings us to the current framework being evaluated against WebdriverIO, which is [Microsoft Playwright](https://github.com/microsoft/playwright), also known as Playwright. The evaluations progress involved the following steps;
+
+- Configuring and Setting up of Wikimedia-core to run on CI, in this case on Github actions.
+- Implementation of all mediawiki-core tests in Playwright
+- Runing mediawiki-core tests in WebdriverIO
+- Analysis of both WebdriverIO and Playwright tests in terms of speed and stability
+
+### Playwright Vs WebdriverIO
+
+[WebdriverIO](https://github.com/webdriverio/webdriverio) is a test automation framework that allows you to run tests based on the Webdriver protocol and Appium automation technology. WebdriverIO is known to be;
+Extendable
+Adding helper functions, or more complicated sets and combinations of existing commands is simple and really useful
+
+Compatible
+WebdriverIO can be run on the WebDriver Protocol for true cross-browser testing as well as Chrome DevTools Protocol for Chromium-based automation using Puppeteer.
+
+Feature Rich
+The huge variety of community plugins allows you to easily integrate and extend your setup to fulfill your requirements.
+
+The above points are just a few of WebdriverIO capability. To explore more WebdriverIO capability, you can check out the [WebdriverIO](https://webdriver.io/) documentation and the [Wikipedia Repo](https://www.mediawiki.org/wiki/Selenium).
+
+[Microsoft Playwright](https://github.com/microsoft/playwright) is a Node.js library to automate Chromium, Firefox, and WebKit with a single API. Playwright is known for the following capability;
+
+Test across all modern browsers
+Single API to automate Chromium, Firefox, and WebKit.
+
+Use in your preferred language
+Use the Playwright API in JavaScript & TypeScript, Python, C# and, Java.
+
+Automate without trade-offs
+Capable automation for single-page apps that rely on the modern web platform.
+
+The above points are just a few capabilities. To explore more on Playwright capability, you can check out the [Playwright](https://playwright.dev/) documentation and this [ forked Wikipedia Repo](https://github.com/hariclerry/mediawiki/tree/master/tests/playwright).
+
+While working with Playwright for the past two months, it has come across as easy to install and setup. The package takes care of installing all the browsers(Chromium, Firefox, and WebKit). Capturing Screenshots is an out-of-the-box experience. However, Video recording requires separate installation of [playwright-video and ffmpeg](https://playwright.tech/blog/record-your-browser-tests-with-playwright) but they all blend in with Playwright seamlessly.
+
+Below are some of the benefits I have experienced and seen while using playwright;
+
+- Console debug option which is very useful in an event when tests are failing anonymously.
```

Comment:
What do you mean by "failing anonymously"?

Reply:
I meant without a clear error messages